tests/playwright/sanity/modules/floating-buttons/floating-element-page.ts

```diff
@@ -0,0 +1,65 @@
+import WpAdminPage from '../../../pages/wp-admin-page';
+import type { APIRequestContext } from '@playwright/test';
+import EditorPage from '../../../pages/editor-page';
+
+export default class FloatingElementPage extends WpAdminPage {
+	async goToFloatingButtonsEmptyPage() {
+		await this.page.goto(
+			'/wp-admin/edit.php?post_type=elementor_library&page=e-floating-buttons',
+		);
+	}
+
+	async goToFloatingButtonsPage() {
+		await this.page.goto(
+			'/wp-admin/edit.php?post_type=e-floating-buttons',
+		);
+	}
+
+	async deleteAllFloatingButtons() {
+		await this.goToFloatingButtonsPage();
+		await this.page.locator( '#cb-select-all-1' ).click();
+		await this.page.locator( '#bulk-action-selector-top' ).selectOption( 'trash' );
+		await this.page.locator( '#doaction' ).click();
+		await this.page.waitForURL( '/wp-admin/edit.php?post_type=e-floating-buttons&paged=1' );
+	}
+
+	async createFloatingButtonWithAPI() {
+		const request: APIRequestContext = this.page.context().request,
+			postDataInitial = {
+				title: 'Playwright Test Floating Element - Uninitialized',
+				status: 'publish',
+				content: '<button aria-controls="e-contact-buttons__content-wrapper" aria-label="Close Floating Element" type="button">\n' +
```

Comment:
@louiswol94 I think we do, not 100% sure though. it is present when inserted from remote.

Reply:
@nicoladj77 @louiswol94 

Maybe we can import this data from a JSON file, to keep this test more readable?

Reply:
@hein-obox so I guess we will move to wp import cli command

Reply:
@nicoladj77 I am not so happy with 'hardcoding' all the post content. We usually generate the post content with the Elementor editor inside or the test. Or we import a json template. I am not sure if you can imports inside the floating editor as well.

E.g. https://github.com/elementor/elementor-pro/blob/bc0076c52c09ff9117497a5113d9161e2c055210/tests/playwright/sanity/modules/off-canvas/off-canvas-a11y.test.ts#L63

Reply:
@hein-obox so the big problem here is that the floating buttons are imported from the remote library, and the remote library is not available in the test environment as we can't connect. And also importing from the remote library would make the test flaky as it would depend on the remote library always returning the same thing. So either we find a way to mock the remote library call or we need to import it. I am ok with either wp import, which is what I am working on, or using a json template, but I do not know how to generate one.